docs/usage/prometheus-grafana.md

```diff
@@ -0,0 +1,22 @@
+**Prometheus**
+* Download Prometheus
+    * https://prometheus.io/download/
+* Unzip the downloaded .zip file
+* Run lodestar beacon node with `--metrics.enabled=true` and `metrics.serverPort=5000` 
+    * *NOTE: 5000 is also the default port specified in the prometheus.yml in the lodestar repo*
+* Run Prometheus from it's installed location with the lodestar prometheus.yml passed in as the config file
+    * `./prometheus --config.file=<lodestar-root-directory>/prometheus.yml`
```

Comment:
Code snippets should be surrounded by triple ticks. See other doc pages

Reply:
fixed

Reply:
Surround block code with triple ticks and new lines like in here. https://github.com/ChainSafe/lodestar/blob/29350cf90a/.codecov.yml
Check the source code or raw to see the actual format.

Reply:
Yes, I put the triple ticks and newlines where appropriate.  Please see the most up-to-date file diff.

Reply:
Also, not really sure what you're trying to point out with that .yml file.

Reply:
Sorry typo, I meant to link this .md https://github.com/ChainSafe/lodestar/blob/29350cf90a/docs/usage/altona.md